### PR TITLE
Rules/linux/auditd

### PR DESCRIPTION
**Pull request by iai-rsa**
### Summary of the Pull Request

- This PR updates a rule in /rules/linux/auditd: _**lnx_auditd_system_info_discovery.yml**_
- It also adds 2 new rules in /rules/linux/auditd:
  - lnx_auditd_unix_shell_configuration_modification.yml   (T1546.004)
  - lnx_auditd_modify_system_firewall.yml (T1562.004)

### Detailed Description of the Pull Request / Additional Comments

- This PR updates the rule _**lnx_auditd_system_info_discovery.yml**_ to detect adversaries doing VM check, environment variables discovery etc.
- It also adds 2 new rules in /rules/linux/auditd.
  - The first new rule (T1546.004) aims to detect unix shell configuration modification. Adversaries may establish persistence through unix shell configuration modification, which may lead to executing malicious commands when a new shell is opened.
  - The second new rule (T1562.004) aims to detect  removal of system firewall rules. Adversaries may only delete or modify a specific system firewall rule to bypass controls limiting network usage or access. Detection rules that match only disabling of firewalls will miss this.

### Example Log Event
NA

### Fixed Issues
NA

### CODE REVIEW

File: rules/linux/auditd/lnx_auditd_unix_shell_configuration_modification.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+title: Unix Shell Configuration Modification
+id: a94cdd87-6c54-4678-a6cc-2814ffe5a13d
+related:
+  - id: e74e15cc-c4b6-4c80-b7eb-dfe49feb7fe9
+    type: obsoletes
+status: experimental
+description: Detect unix shell configuration modification. Adversaries may establish persistence through executing malicious commands triggered when a new shell is opened.
+references:
+    - 'MITRE Attack technique T1546.004; https://attack.mitre.org/techniques/T1546/004/'
```
Comment (phantinuss) [inline rules/linux/auditd/lnx_auditd_unix_shell_configuration_modification.yml:9]:
```suggestion
    - 'https://attack.mitre.org/techniques/T1546/004/'
```

File: rules/linux/auditd/lnx_auditd_unix_shell_configuration_modification.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+logsource:
+    product: linux
+    service: auditd
+detection:
+    selection:
+        type: 'PATH'
+        name:
+            - '/etc/profile'
+            - '/etc/profile.d/*'
+            - '/etc/bash.bashrc'
```
Comment (phantinuss) [inline rules/linux/auditd/lnx_auditd_unix_shell_configuration_modification.yml:25]: Is there a reason you didn't include:
```
            - '/etc/shells'
            - '/etc/bashrc'
            - '/etc/csh.cshrc'
            - '/etc/csh.login'
```
?

File: rules/linux/auditd/lnx_auditd_unix_shell_configuration_modification.yml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+related:
+  - id: e74e15cc-c4b6-4c80-b7eb-dfe49feb7fe9
+    type: obsoletes
+status: experimental
+description: Detect unix shell configuration modification. Adversaries may establish persistence through executing malicious commands triggered when a new shell is opened.
+references:
+    - 'MITRE Attack technique T1546.004; https://attack.mitre.org/techniques/T1546/004/'
+author: Peter Matkovski, IAI 
+date: 2023/03/06
+modified: 2023/03/26
```
Comment (phantinuss) [inline rules/linux/auditd/lnx_auditd_unix_shell_configuration_modification.yml:12]: living in the future? :)
```suggestion
```

**Review by phantinuss — APPROVED** [review]

**Comment by frack113** [comment]
Hi, Why remove [rules/linux/auditd/lnx_auditd_alter_bash_profile.yml](https://github.com/SigmaHQ/sigma/pull/4095/files#diff-0ad5b8fc853e6941103c7444f96588fdc3e0e0b950feaa6673f1c939f993abac) ?

**Pull request merged**